components/sidebar/sidebar_header/dropdown/sidebar_header_dropdown_button.tsx

```diff
@@ -111,9 +104,9 @@ export default class SidebarHeaderDropdownButton extends React.PureComponent<Pro
                     id='headerInfo'
                     className='header__info'
                 >
-                    <VerticalStack>
-                        {teamNameWithToolTip}
-                        {!this.props.globalHeaderEnabled &&
+                    {this.props.globalHeaderEnabled ? teamNameWithToolTip : (
```

Comment:
Apologies Michel, I missed this in my first quick scan. The channel header has been addressed in the [Channel Header PR](https://github.com/mattermost/mattermost-webapp/pull/8569) as well.

Reply:
I just now reverted the changes